tests/build_tests/super_errors/expected/optional_fn_argument_pass_option.res.expected

```diff
@@ -0,0 +1,18 @@
+
+  [1;31mWe've found a bug for you![0m
+  [36m/.../fixtures/optional_fn_argument_pass_option.res[0m:[2m5:18[0m
+
+  3 [2m│[0m let t = Some(1)
+  4 [2m│[0m 
+  [1;31m5[0m [2m│[0m let f = optFn(~x=[1;31mt[0m)
+  6 [2m│[0m 
+
+  This has type: [1;31moption<int>[0m
+  But this optional function argument [1;33m~x[0m is expecting: [1;33mint[0m
+  
+  You're passing an optional value into an optional function argument.
+  Optional function arguments expect you to pass the concrete value, not an option.
```

Comment:
nitpicking again, but I'm not a big fan of concrete here, especially since sometimes, the concrete value that is expected can actually be an option, maybe use "unwrapped" instead? I won't start a war over this, it's really ok as is too.